tutorindigo/plugin.py

```diff
@@ -204,3 +165,49 @@ def _override_openedx_docker_image(
         ),
     ]
 )
+
+
+# Apply patches from tutor-indigo
+for path in glob(
+    os.path.join(
+        str(importlib_resources.files("tutorindigo") / "patches"),
+        "*",
+    )
+):
+    with open(path, encoding="utf-8") as patch_file:
+        hooks.Filters.ENV_PATCHES.add_item(
+            (os.path.basename(path), patch_file.read())
+        )
+
+
+for mfe in indigo_styled_mfes:
+    PLUGIN_SLOTS.add_items([
```

Comment:
```suggestion
    PLUGIN_SLOTS.add_item(
```

If you want to avoid the need for the `[]`, you can use the singular form.